clients/mediaconvert_test.go

```diff
@@ -33,22 +36,36 @@ func TestOnlyS3URLsToAWSClient(t *testing.T) {
 			return nil, errors.New("secret error")
 		},
 	}
-	mc, f, transferDir, cleanup := setupTestMediaConvert(t, awsStub)
+	mc, f, _, cleanup := setupTestMediaConvert(t, awsStub)
 	defer cleanup()
+	sz, err := f.Stat()
+	require.NoError(err)
 
-	err := mc.Transcode(context.Background(), TranscodeJobArgs{
+	err = mc.Transcode(context.Background(), TranscodeJobArgs{
 		InputFile:     mustParseURL(t, "file://"+f.Name()),
 		HLSOutputFile: mustParseURL(t, "s3+https://endpoint.com/bucket/1234/index.m3u8"),
 		CollectSourceSize: func(size int64) {
-			require.Equal(len(exampleFileContents), int(size))
+			require.Equal(sz.Size(), int64(size))
 		},
 	})
 	require.ErrorContains(err, "secret error")
-
-	// Check that the file was copied to the osTransferBucketURL folder
-	content, err := os.ReadFile(path.Join(transferDir, "input/1234/video"))
-	require.NoError(err)
-	require.Equal(exampleFileContents, string(content))
+	/*
```

Comment:
@mjh1: FYI - mind fixing this? I added md5 checks on the files since we're using a mp4 file now but keep getting md5 mismatches.

Reply:
Reading the input file `f` at this point in the code resulted in all zeros, I guess because it has already been read. I fixed it by re-opening the file 👍